test/indexed_test.dart

```diff
@@ -177,4 +201,59 @@ void main([List<String>? args]) {
     compareSpeed('objectId', objectIdFactory);
     compareSpeed('uuid', uuidFactory);
   });
+
+  final testCases = [
+    FtsTestData('lord of the', <String>{lordOfTheFlies, lordOfTheRings, wheelOfTime, silmarillion}),
+    FtsTestData('fantasy novel', <String>{lordOfTheRings, wheelOfTime}),
+    FtsTestData('popular english', <String>{lordOfTheFlies, silmarillion}),
+    FtsTestData('amazing awesome stuff', <String>{}),
+    FtsTestData('fantasy -novel', <String>{silmarillion}),
+  ];
+
+  Realm setupFtsTest() {
+    final config = Configuration.local([ObjectWithFTSIndex.schema]);
+    final realm = getRealm(config);
+
+    ObjectWithFTSIndex addObject(String title, String summary) {
+      final value = realm.add(ObjectWithFTSIndex(title, summary));
+      value.nullableSummary = summary;
+      return value;
+    }
+
+    realm.write(() {
+      final objectWithDefaults = addObject('N/A', '');
+      objectWithDefaults.nullableSummary = null;
+
+      addObject(animalFarm,
```

Comment:
it is educational, but do we really need so much text to test FTS

Reply:
I was too lazy to write new tests and copied the ones from the .NET SDK. They did reveal several bugs with the original implementation, so I'd say they were useful at the time. I'm not sure if they continue to be useful and probably a more compact search text would be just fine. That being said, it'll be a little bit extra effort to come up with equivalent tests and I'm not sure the benefit is there.

Reply:
ok lets copy another malpractice from the .NET SDK :) At least it will make everyone know what `silmarillion` is.

Reply:
The .NET SDK: educating developers since 2015 😜